tests/logger/test_logger.py

```diff
@@ -450,3 +451,240 @@ def test_inject_train_speed_fault(
         assert log_entry.track_speed_limit_fault_configuration is None
         assert log_entry.schedule_blocked_fault_configuration is None
         assert log_entry.train_prio_fault_configuration is None
+
+    @freeze_time()
+    def resolve_platform_blocked_fault(
+        self,
+        run,
+        tick,
+        platform_blocked_fault_configuration,
+    ):
+        logger = Logger(run_id=run.id)
+        logger.resolve_platform_blocked_fault(
+            tick=tick,
+            platform_blocked_fault_configuration=platform_blocked_fault_configuration,
+        )
+        log_entry = (
+            ResolveFaultLogEntry.select()
+            .where(
+                ResolveFaultLogEntry.tick == tick
+                and ResolveFaultLogEntry.platform_blocked_fault_configuration
+                == platform_blocked_fault_configuration
+            )
+            .first()
+        )
+        assert log_entry.timestamp == datetime.now()
+        assert log_entry.tick == tick
+        assert (
+            log_entry.message
+            == f"Platform blocked fault with configuration {platform_blocked_fault_configuration} "
+            f"resolved"
+        )
+        assert log_entry.run_id.id == run.id
+        assert (
+            log_entry.platform_blocked_fault_configuration
+            == platform_blocked_fault_configuration
+        )
+
+        assert log_entry.track_blocked_fault_configuration is None
+        assert log_entry.track_speed_limit_fault_configuration is None
+        assert log_entry.schedule_blocked_fault_configuration is None
+        assert log_entry.train_prio_fault_configuration is None
+        assert log_entry.train_speed_fault_configuration is None
+
+    @freeze_time()
+    def resolve_track_blocked_fault(
+        self,
+        run,
+        tick,
+        track_blocked_fault_configuration,
+    ):
+        logger = Logger(run_id=run.id)
+        logger.resolve_track_blocked_fault(
+            tick=tick,
+            track_blocked_fault_configuration=track_blocked_fault_configuration,
+        )
+        log_entry = (
+            ResolveFaultLogEntry.select()
+            .where(
+                ResolveFaultLogEntry.tick == tick
+                and ResolveFaultLogEntry.track_blocked_fault_configuration
+                == track_blocked_fault_configuration
+            )
+            .first()
+        )
+        assert log_entry.timestamp == datetime.now()
+        assert log_entry.tick == tick
+        assert (
+            log_entry.message
+            == f"Track blocked fault with configuration {track_blocked_fault_configuration} "
+            f"resolved"
+        )
+        assert log_entry.run_id.id == run.id
+        assert (
+            log_entry.track_blocked_fault_configuration
+            == track_blocked_fault_configuration
+        )
+
+        assert log_entry.platform_blocked_fault_configuration is None
```

Comment:
Love you tests, because you test that the other configuration fields are empty ❤️